app/root.css

```diff
@@ -257,24 +247,23 @@ article > h2 > .transparent-button {
 }
 
 article > h2::after {
-  content: '›';
-  margin-top: -8px;
-  font-size: 2em;
-  line-height: 0;
-  font-weight: normal;
+  content: url('/assets/chevron-right.svg');
+  width: 12px;
+  height: 12px;
+  filter: var(--light, invert(0)) var(--dark, invert(1));
   transition: transform 400ms;
```

Comment:
The chevron size doesn't stay 12x12. It shrinks when the screen is small and questions wrap onto the 2nd line. How would you recommend fixing? Should I style it in JSX instead of using CSS ::after?
<img width="117" alt="Screen Shot 2022-07-03 at 1 36 28 PM" src="https://user-images.githubusercontent.com/38997995/177056453-0d6ca2b4-1e4d-4acc-b1a5-f91243c1e358.png">

Reply:
if the parent `h2` element is still `display: flex`, we need to add `flex: none;` to the chevron to stop it from shrinking - https://developer.mozilla.org/en-US/docs/Web/CSS/flex#values

Reply:
Ah, that was easy! I knew you'd know off the top of your head without me digging around.